config.py

```diff
@@ -1,8 +1,14 @@
 import json
 import os
+import logging
 from typing import List
 from dataclasses import dataclass
 
+logging.basicConfig(level=logging.INFO,
+                    filemode='a',
+                    format='%(asctime)s %(name)s %(levelname)s %(message)s',
+                    datefmt='%H:%M:%S',
+                    filename='/etc/tinyHIPPO/tinyHIPPO_error.log')
```

Comment:
Does this create another config variable?

Reply:
This just instantiates a logging object which is saved in `self.log_event = logging.getLogger()` so that any class or file can just get the logging object and call it appropriately.

Reply:
Okay, if it's a static value can it's declaration be moved to the constructor for the Config object?